File: Python/Ejercicios-LOO/ejercicio-15.py
# 15. Dado un conjunto de valores numéricos indicar cuál es el mayor. El ingreso de datos finaliza con la llegada de un cero.

num = int(input("Ingrese un número: "));
max_num = 0;
while num != 0:
    if num > max_num:
        max_num = num;
    num = int(input("Ingrese otro número: "));
        
print(f"El mayor número inresado fue: {max_num}");

File: Java/src/EjerciciosLOO/Ejercicio_15.java
package EjerciciosLOO;

import java.util.Scanner;

public class Ejercicio_15 {
    public static void main(String[] args) {
        // Dado un conjunto de valores numéricos indicar cuál es el mayor. El ingreso de
        // datos finaliza con la llegada de un cero.

        Scanner sc = new Scanner(System.in);

        System.out.println("Ingrese un número:");
        int n = sc.nextInt();

        int mayor = 0;

        while (n != 0) {
            if (n > mayor) {
                mayor = n;
            }

            System.out.println("Ingrese otro número:");
            n = sc.nextInt();
        }

        sc.close();

        System.out.println("El mayor número ingresado fue: " + mayor);
    
    }
}
